src/Client/AzureManaged/DurableTaskSchedulerClientExtensions.cs

```diff
@@ -40,15 +40,70 @@ public static void UseDurableTaskScheduler(
             configure);
     }
 
+    /// <summary>
+    /// Configures Durable Task client to use the Azure Durable Task Scheduler service.
+    /// </summary>
+    /// <param name="builder">The Durable Task client builder to configure.</param>
+    /// <param name="endpointAddress">The endpoint address of the Durable Task Scheduler resource. Expected to be in the format "https://{scheduler-name}.{region}.durabletask.io".</param>
+    /// <param name="taskHubName">The name of the task hub resource associated with the Durable Task Scheduler resource.</param>
+    /// <param name="credential">The credential used to authenticate with the Durable Task Scheduler task hub resource.</param>
+    /// <param name="retryOptions">The options that determine how and when a request will be retried.</param>
+    /// <param name="configure">Optional callback to dynamically configure DurableTaskSchedulerClientOptions.</param>
+    public static void UseDurableTaskScheduler(
+        this IDurableTaskClientBuilder builder,
+        string endpointAddress,
+        string taskHubName,
+        TokenCredential credential,
+        DurableTaskSchedulerClientOptions.ClientRetryOptions retryOptions,
```

Comment:
Rather than creating a new overload with a dedicated parameter for `retryOptions`, customers should just use the existing overload above and specify retry options in the `configure` action.

Reply:
I was following what I thought was the convention of adding extra constructors for ease of use. If we don't need that, happy to use the `configure`.

Reply:
Adding extra constructors in most cases is only for preserving backwards binary compatibility. In general, it's better to have fewer APIs (fewer things to document) so we try to add new overloads only when strictly necessary.